.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+# Pre-commit configuration file,
+# when Streamlit's pre-commit detects that one of the linters has failed,
+# it automatically lints the files and does not allow the commit to pass. 
+# Please review the changes after lint has failed and commit them again,
+# the second commit should pass,
+# because the files were linted after trying to do the first commit.
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+      - id: black
+        args:
+          # Configure Black to support only syntax supported by the minimum supported Python version in setup.py.
+          - --target-version=py36
```

Comment:
nit: I think our min python version now is 3.7 (the version in the Makefile must have been set when it was min 3.6)

Reply:
Thank you, it's changed 👍